horace_core/sqw/file_io/@dnd_binfile_common/dnd_binfile_common.m

```diff
@@ -474,18 +499,21 @@ function check_obj_initated_properly(obj)
             obj.file_id_ = 0;
         end
         %
-        function obj = activate(obj)
+        function obj = activate(obj, permission)
```

Comment:
no documentation for the `permission` argument... what is that? what is the default?

Reply:
if we're only ever opening in a finite set of modes would adding those string to the class as well named constants be helpful so you could call `.activate(READ_ONLY)` ?